sdjournal/journal.go

```diff
@@ -237,3 +237,17 @@ func (j *Journal) Wait(timeout time.Duration) int {
 
 	return int(r)
 }
+
+// Returns the journal disk space usage, in bytes.
```

Comment:
Please update the docstring to be consistent with the rest:
GetUsage returns the journal disk space usage, in bytes.